Parsers/TrendMicro/TrendMicroTippingPoint

```diff
@@ -0,0 +1,15 @@
+CommonSecurityLog
```

Comment:
Could you please include steps required for setting up the parser? Please refer description, usage and references of a parser for a sample here https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/SymantecVIP/SymantecVIP.txt.

Reply:
Included the steps requested. Thanks for the suggestion.